site/book/03-packages/04-updating-a-package.md

```diff
@@ -0,0 +1,48 @@
+Since a forked package contains a reference to its upstream package, kpt can fetch any update to the
+upstream package and merge it with local changes.
+
+The package can be updated with changes from upstream in a declarative way by updating the
+`upstream` section in the `Kptfile` then running the `update` command.
+
+For example, change the value of `upstream.git.ref` to `v0.2`:
```

Comment:
Not a blocker but I'd go with the CLI update first.  In other package managers the file format is a black box and most of the people probably expect to update with the CLI first.